b2g/chrome/content/remote_control.js

```diff
@@ -72,5 +72,5 @@ const kPrefPresentationDiscoverable = "dom.presentation.discoverable";
     Services.obs.addObserver(networkStatusMonitor, "network:offline-status-changed", false);
   }
 
-  setupRemoteControl();
+  window.onmoztimechange = setupRemoteControl;
```

Comment:
This patch is too specific to the boot sequence of Nexus player. What we should do here is adding a logic of:
1. RemoteControlService.stop() if service is running.
2. delete the previous generated key
3. RemoteControlService.start() if necessary.